funqy-quickstarts/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,54 @@
+####
```

Comment:
This one slipped though (there is a typo in your script; `funky` -> `funqy`).

Reply:
Raaaaaah. I fixed it manually first and then automated it but did it wrong :).

Reply:
Fixed!